bin/crew

```diff
@@ -1268,6 +1268,23 @@ def shrink_dir(dir)
   end
 end
 
+def install_files(prefix, path, dst = File.join( CREW_PREFIX, src.delete_prefix(".#{CREW_PREFIX}") ))
+  src = File.join(prefix, path)
```

Comment:
You are declaring src inside the function, so you can't use it in the argument.

However, this doesn't work:
```
def install_files(prefix, path, dst = File.join( CREW_PREFIX, (File.join(prefix, path)).delete_prefix(".#{CREW_PREFIX}") ))
  src = File.join(prefix, path)
  puts "dst is #{dst} and src is #{src}".lightblue
  return false unless Dir.exist?(src)
```

I get this: 
![image](https://user-images.githubusercontent.com/1096701/222265152-f96fbf64-8f8b-44aa-9183-b7cb0f93c2ea.png)

Reply:
This works:
`def install_files(prefix, path, dst = File.join( CREW_PREFIX, (File.join(prefix, path)).delete_prefix(".#{CREW_PREFIX}") ))`